runtime/battery-station/src/parameters.rs

```diff
@@ -150,7 +153,7 @@ parameter_types! {
     /// Maximum Categories a prediciton market can have (excluding base asset).
     pub const MaxCategories: u16 = MAX_CATEGORIES;
     /// Maximum number of disputes.
-    pub const MaxDisputes: u16 = 6;
+    pub const MaxDisputes: u16 = 3;
```

Comment:
I should highlight my change here. Is this reasonable to test the global dispute functionality more often on the test-net?

Reply:
I think even that would trigger the global dispute functionality seldom.

Reply:
Because `Court` is better way to decide for the truth than just trusting the ones with the most `ZTG`.